pkg/logs/input/file/tailer.go

```diff
@@ -117,6 +119,15 @@ func (t *Tailer) setup(offset int64, whence int) error {
 	return nil
 }
 
+// buildTailerTags groups the file tag, directory (if wildcard path) and user tags
+func (t *Tailer) buildTailerTags() []string {
```

Comment:
should we also add `filepath` just in case ?

Reply:
For that, I've checked with Nils and no need for now. (dir name and file name are both available)